src/components/Autocomplete/Autocomplete.st.css

```diff
@@ -46,6 +51,19 @@
     align-items: center;
 }
 
+DropdownContent::dropdownOption {
```

Comment:
replace with `.root::dropdownContent`

why it works:
`.root` extends `CoreAutocomplete`
`CoreAutocomplete` extends `InputWithOptions` 
`InputWithOptions` extends `Dropdown`
`Dropdown` exposes `dropdownContent` class which overrides the dropdown content

why your solution also work:
it's like targeting a tag name. it works because there is a dropdown, but theoretically, the implementation can change and not be the dropdown from core, but something else. maintaining the CoreAutocomplete API preserves the behavior.